### ISSUE TITLE
upgrade feed-io dependency

### ISSUE THREAD

**Issue by lesion** [OleVik/grav-plugin-twigfeeds#30]
hi there! I'm using this plugin and trying to get enclosures / images from feeds.
In my case this is possible by upgrading  `feed-io` dependency from **4.3** to **5.0.4** (the latest).
Are you interested in accepting a PR for this? Unfortunately this would also change the required php version >=8.0

**Comment by OleVik** [comment]
Hi,

Because Grav still supports PHP 7.3, continued support for it is necessary. However, PHP 7.4 is already beyond active support, and will stop receiving security fixes after November 2022. I would welcome a pull request for it, bumping feed-io to v5, that can be released as a beta-version.

Even if Grav doesn't drop support for PHP 7.4 towards the end of the year, I am fine with implementing the requirement at that point and letting attempted installations with lower versions fail gracefully - suggesting the installation of a lower version.

**Comment by hughbris** [comment]
I'm a bit lost but think the problem I experienced deploying `master` might be related to this update. So adding here rather than opening a new issue at first.

I got a composer error message indicating I need PHP 8:

```
Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.0.2". You are running 7.4.27.
```

This issue discussion helped me figure out that a downgrade might be worth trying, and indeed 4.0.1 works.

However, I think requiring 8 is a little premature and the recommended downgrade version could perhaps have a stable tag label.

My 2c on the experience, but I'm happy enough as I have a working version :)

**Comment by OleVik** [comment]
It is, and it is why 5.0.0-beta.1 is a pre-release. Of course, I forgot the `testing: true` parameter in  blueprints.yaml, because it is such an anti-pattern that it doesn't even cross my mind. This is resolved in [5.0.0-beta.2](https://github.com/OleVik/grav-plugin-twigfeeds/releases/tag/v5.0.0-beta.2).

PHP 7 support will be maintained, and PHP 8 deferred, until the end of November 2022 - per PHP's [supported versions cycle](https://www.php.net/supported-versions.php). This is not strictly to follow PHP's cycle, but because the underlying library - FeedIO - requires PHP 8 to be upgraded to the newest version.

Whether Grav will follow the same cycle remains to be seen, but at the end of the year the error you experienced is exactly what should happen to prevent an incompatibility between the server-environment and an update of the plugin.

**Comment by hughbris** [comment]
Thanks. I forgot to consider whether I originally cloned your repo or grabbed it through `gpm` and I don't remember now.
